examples/stm32u0/.cargo/config.toml

```
@@ -0,0 +1,9 @@
+[target.'cfg(all(target_arch = "arm", target_os = "none"))']
+# replace STM32G071C8Rx with your chip as listed in `probe-rs chip list`
+runner = "probe-rs run --speed 100 --chip STM32u083rctx"
```

```suggestion
runner = "probe-rs run --chip STM32u083rctx"
```

Not sure if you found you needed to limit the speed, but it works fine for me without (and it's almost 10x the speed)

---

ahh tha'ts why it flashed so slow! :rofl:  (it's copypasted from the c0 examples...)